Mlem/App/Views/Root/Tabs/Settings/InteractionBarEditor/InteractionBarWidgetPickerView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  InteractionBarWidgetPickerView.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2025-02-12.
+//
+
+import SwiftUI
+
+struct InteractionBarWidgetPickerView<Configuration: InteractionBarConfiguration>: View {
+    @Environment(Palette.self) var palette
+    
+    @Binding var configuration: Configuration
+    
+    var body: some View {
+        Form {
+            Section {
+                Text("Select which widgets to display in your palette")
```

Comment:
This should have a full stop I think. Also maybe "Choose" is better than "Select"? I'm not sure
```suggestion
                Text("Choose which widgets to display in your palette.")
```